packages/thread-context/tests/bridge-test.js

```
@@ -0,0 +1,124 @@
+import { createThreadContext, BridgeError, BridgeClient, createCollectionProxy, createMethodProxy, getActiveBridgeCount } from 'meteor/thread-context';
```

ESLint

[error] Replace `·createThreadContext,·BridgeError,·BridgeClient,·createCollectionProxy,·createMethodProxy,·getActiveBridgeCount·` with `⏎··createThreadContext,⏎··BridgeError,⏎··BridgeClient,⏎··createCollectionProxy,⏎··createMethodProxy,⏎··getActiveBridgeCount,⏎`

(prettier/prettier)